### ISSUE TITLE
[Feature Request] Recommendations to friends and knowing when media is completely consumed

### ISSUE THREAD

**Issue by nomandera** [Fallenbagel/jellyseerr#211]
### Description

Managing content suggestions and retirements within my family library is a chore. Suggesting shows to watch and tracking if they are completely viewed, in progress, abandoned or blocked by age restrictions is more effort than it should be.

Being able to suggest a show to someone via JellySeer rather than hollering to go read a synopsis whilst you wait would save a lot of effort.

Having that user then be able to tag their interest level, especially if they don't want to watch it, solves the dilemma that watch status alone cannot determine if media has been consumed i.e. "have watched the entire show", "will never watch a show" and "are not old enough to watch a show" are all logically the same, the show is completed for this user.

### Desired Behavior

- Be able to suggest a media to someone via JellySeer
- Be able to track interest level of suggestions
- Be able to summarise status for my family to make deletion and archival decisions

### Additional Context

I realise this is possibly out of scope however JellySeer is in a unique "overwatch" position to do this especially since the concept of recommendations already exists.

### Code of Conduct

- [X] I agree to follow Overseerr's Code of Conduct

**Comment by Fallenbagel** [comment]
Do you mean kind of like a upvote system?

**Comment by nomandera** [comment]
Sorry for the wall of words. :)

Whilst trying to think about how to answer this other than just agree that it is a sort of "upvote system" I thought a real world example would help set the scene. 



I manage the content for my close family using Emby to offer a single contiguous interface and some semblance of privacy. No one can host all things for all time so I need to make informed decisions on what can be archived or deleted and when.

Data comes into the system via Jellyseerr requests and other normal methods. This is where it starts to get interesting.

If a user has requested something then I know it is wanted, however I have no idea if the rest of my family are interested in it. This means I never have complete information to make any definitive archival or deletion decisions unless I just guess or actually talk to everyone for each specific bit of content.

The traditional approach to this is to use watch status as an indicator of completion but it is very rare with a 70 year difference in age range content is ever completely ticked watched. It gets even more complicated when some of the content cannot be consumed as the kids are too young for it or maybe someone wants to watch the content but not start until the season is complete.

There are many more variations in here such as someone starts one episode of a show to see if they like it and this partial watch state could either mean they loved it and are waiting for a good time to watch or they hated it and will never watch.


This is the crux of the problem and the suggestion. Jellyseerr already has the concept of "recommended" but it is single user algorithm driven. Jellyseerr is also in a unique position where it "stands above" the media library and could help the system admin collect and summarise information to help in the archival and deletion decision process.

So for example:

- "The system admin wants to know if you are plan to watch this Season"
- "User X has recommended this show just for you, are you going to watch it? Now-Later-Never-Dont know yet" 
- "User X has recommended this show for the entire family" 
- "Did you like this show? Yes-Will never watch" 
- "The system will remove this content in XX days. Request deletion delay yes/no"

I have other ideas for this but they are all variations on the theme, Jellyseerr through automation and direct user interactions summarising content status.
